keyboards/crkbd/keymaps/mvi/keymap.c

```diff
@@ -0,0 +1,909 @@
+                                         _______, _______, KC_SPC,      _______, _______, _______
+                                      //`--------------------------'  `--------------------------'
+  ),
+
+  [_MATHS] = LAYOUT(
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+     MA_NAT,  MA_LDB,  MA_RDB,  SUB_P,   SUB_O,   MA_NIN,                       _______, SUB_V,   MA_SCP,  SUB_L,   SUB_J,   _______,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+     MA_REL,  SUB_A,   SUB_U,   SUB_I,   SUB_E,   MA_INI,                       MA_CPL,  SUB_T,   SUB_S,   SUB_R,   SUB_N,   SUB_M,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+     MA_REE,  _______, _______, SUB_X,   _______, SUB_K,                        _______, MA_RAT,  _______, SUB_H,   _______, _______,
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                         _______, _______, _______,    _______, _______, _______
+                                      //`--------------------------'  `--------------------------'
+  ),
+
+  [_ADJUST] = LAYOUT(
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+     RESET,   MVI_CLK, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, KC_DEL ,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+     KC_ESC,  _______, _______, _______, _______, _______,                      _______, BEPO,    NUMPAD,  _______, _______, DEBUG,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+     _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                         _______, _______, _______,    _______, _______, _______
+                                      //`--------------------------'  `--------------------------'
+  )
+};
+
+int RGB_current_mode;
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+// Setting ADJUST layer RGB back to default
+void update_tri_layer_RGB(uint8_t layer1, uint8_t layer2, uint8_t layer3) {
+  if (IS_LAYER_ON(layer1) && IS_LAYER_ON(layer2)) {
+    layer_on(layer3);
+  } else {
+    layer_off(layer3);
+  }
+} 
+
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Same with the tri layer function 
```suggestion
```